lib/types.nix

```diff
@@ -216,19 +250,22 @@ rec {
     };
 
     unspecified = mkOptionType {
-      name = "unspecified";
+      name = "unspecified value";
+      descriptionClass = "noun";
```

Comment:
This isn't right, the `name` shouldn't be changed, but instead a `description` should be added

Reply:
Doesn't seem to lead to a problem, but here's the cleanup #192755